lib/tasks/grda_warehouse.rake

```diff
@@ -282,6 +282,13 @@ namespace :grda_warehouse do
   desc 'Full import routine'
   task daily: [:environment, 'log:info_to_stdout'] do
     Importing::RunDailyImportsJob.new.perform
+
+    HmisSupplemental::DataSet.order(:id).each do |data_set|
+      HmisSupplemental::ImportJob.new.perform(data_set_id: data_set.id)
+    rescue StandardError => e
+      puts e.message
+      Sentry.capture_exception(e)
+    end
```

Comment:
This should probably go in the `hourly` rake task and only execute in the set hour.  See `MaReports::CsgEngage::Report.run_if_ready` for a decent example.  It might mean adding an `hour` column to the data set, or in the `additional_headers` column for the credential.  That way we can configure which hour per installation, and it isn't dependent on a, multi-hour process to finish successfully before it runs.

Reply:
Does the hour need to be configurable? Looks like we're using hard-coded hours